base/dump-dotnet2/dumpdotnet2.csproj

```diff
@@ -0,0 +1,19 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
```

Comment:
I added the assembly name just to be more explicit

```xml
<AssemblyName>dumpdotnet2</AssemblyName>
```

Not a huge issue, but just a thought.